<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectBuilder</class>
 <widget class="QDialog" name="ProjectBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Project Build</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Main Window Class Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leMainWindow">
     <property name="text">
      <string>ColoUiWindow</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>QtCreator Project Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leProjectName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>QtCreator Project Location</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="leProjectLocation"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbSearchPLoc">
       <property name="text">
        <string>Search ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>ColoUi Folder Location</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leColoUiFolderLocation"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbSearch">
       <property name="text">
        <string>Search ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbBuild">
       <property name="text">
        <string>Build</string>
       </property>
       <property name="icon">
        <iconset resource="assets.qrc">
         <normaloff>:/assets/qt.png</normaloff>:/assets/qt.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lMessage">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
